device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttIotHubConnection.java

```diff
@@ -159,7 +159,7 @@ else if (this.config.getAuthenticationType() == DeviceClientConfig.AuthType.X509
                 }
                 else
                 {
-                    serviceParams = TransportUtils.IOTHUB_API_VERSION_PREVIEW + "&" + ModelIdParam + "=" + modelId;
```

Comment:
Why are you switching from a preview API version to a normal API version? This code is going into a preview branch